### PR TITLE
[2-15] Test Page Component

### PR DESCRIPTION
**Pull request by boburwebdev**
Added test page

### CODE REVIEW

**Comment by YanaBr** [comment]
Please, change base branch to 'dev' by clicking on Edit button

**Comment by KotdenGit** [comment]
mat-align-tabs= This thing is not used here. Don't need it?

**Review by KotdenGit — APPROVED** [review]

**Review by YanaBr — CHANGES_REQUESTED** [review]
Please, rename component, add module for it and perform cleanup

**Comment by KuznetsovR** [comment]
Why so many rows were added to the package-lock?

**Comment by ozhurova** [comment]
Why the name is task  [2-5]? It's name the Footer task

File: exalearn-ui/src/app/app.module.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { MainPageModule } from './pages/main-page/main-page.module';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```
Comment (YanaBr) [inline exalearn-ui/src/app/app.module.ts]: Do we need it inside appModule? Maybe move it under your testPageModule?

File: exalearn-ui/src/app/components/timer/timer.component.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	position: fixed;
+	bottom: 0;
+	left: 0;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	z-index: 10;
+
+	div {
+		font-size: 20px;
```
Comment (YanaBr) [inline exalearn-ui/src/app/components/timer/timer.component.scss]: Really bad practice to use tags as a selector. Please, add specific class to this div tag and use it here

Comment (YanaBr) [inline exalearn-ui/src/app/components/timer/timer.component.scss]: Check if we have font constant suitable to this situation (and then you can import just base.scss file instead colors and fonts

File: exalearn-ui/src/app/components/timer/timer.component.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+
+// Test time is set to 1 hour (in seconds)
+const TEST_DURATION = 3600;
```
Comment (YanaBr) [inline exalearn-ui/src/app/components/timer/timer.component.ts:5]: We need to make component flexible and give the ability to set the time duration through a custom attribute
(Just put at least TODO: ... to make it later)

File: exalearn-ui/src/app/components/timer/timer.component.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+	constructor() {}
+
+	ngOnInit(): void {
+		this.startTimer();
+	}
+
+	startTimer() {
+		let timer = TEST_DURATION;
+
+		const displayTimeLeft = () => {
```
Comment (YanaBr) [inline exalearn-ui/src/app/components/timer/timer.component.ts]: Please, move displayTimeLeft to the separate function

File: exalearn-ui/src/app/pages/main-page/main-page.component.scss [hunk 5]
```diff
@@ -52,4 +52,12 @@
 	min-width: 300px;
 	height: 8vh;
 	margin: 10px 0 10px;
+	a {
+		color: $white;
```
Comment (YanaBr) [inline exalearn-ui/src/app/pages/main-page/main-page.component.scss:56]: Same as above, set up specific class to such `<a>` tag (for instance 'routing-link', but it's up to you)

File: exalearn-ui/src/app/app.module.ts (hunk 1 above)
Comment (boburwebdev) [reply exalearn-ui/src/app/app.module.ts]: I'll move it

File: exalearn-ui/src/app/components/timer/timer.component.scss (hunk 2 above)
Comment (boburwebdev) [reply exalearn-ui/src/app/components/timer/timer.component.scss]: Got it

Comment (boburwebdev) [reply exalearn-ui/src/app/components/timer/timer.component.scss]: Got it

**Review by YanaBr — APPROVED** [review]

**Pull request merged**